app/server/models/credit_transfer.py

```diff
@@ -179,11 +187,10 @@ def _get_required_prior_tasks(self):
         For why this works, see https://github.com/teamsempo/SempoBlockchain/pull/262
 
         """
-
         # We're constantly querying complete transfers here. Lazy and DRY
         complete_transfer_base_query = (
             CreditTransfer.query.filter(CreditTransfer.transfer_status == TransferStatusEnum.COMPLETE)
-        )
+        ).execution_options(show_all=True)
```

Comment:
I realized that running `_get_required_prior_tasks` before the end of the request was MUCH slower than after. Turns out that after request,  the `before_compile` event listener doesn't listen any more, and that it was slowing these particular queries WAY down before adding these `execution_options`.